package.json

```diff
@@ -59,16 +72,19 @@
     "redux-form": "^7.0.3",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.2.0",
+    "sinon": "^4.1.2",
     "uglifyjs-webpack-plugin": "^1.0.0-beta.3",
     "webpack": "^3.5.5",
     "webpack-merge": "^4.1.0"
   },
   "devDependencies": {
     "babel-eslint": "^8.0.0",
+    "babel-jest": "^21.2.0",
     "babel-loader": "^7.1.2",
     "babel-plugin-transform-async-to-generator": "^6.24.1",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-preset-env": "^1.6.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
We already use ```babel-preset-env```, so it installs plugins we need to support last ECMAScript release. Could you please remove es2015